pkg/perf/publicip/publicip_task.go

```diff
@@ -62,6 +63,7 @@ func NewTask() perf.Task {
 		taskID:      taskID,
 		schedule:    taskSchedule,
 		description: taskDescription,
+		jitter:      10 * 60,
```

Comment:
Why this has to be defined as an attribute? you can always return this in the jitter function itself.